optimize/backend/src/main/java/io/camunda/optimize/license/LicenseType.java

```diff
@@ -0,0 +1,41 @@
+/*
+ * Copyright Camunda Services GmbH and/or licensed to Camunda Services GmbH under
+ * one or more contributor license agreements. See the NOTICE file distributed
+ * with this work for additional information regarding copyright ownership.
+ * Licensed under the Camunda License 1.0. You may not use this file
+ * except in compliance with the Camunda License 1.0.
+ */
+package io.camunda.optimize.license;
+
+import java.util.Collections;
+import java.util.HashMap;
+import java.util.Map;
+
+public enum LicenseType {
+  SAAS("saas"),
+  PRODUCTION("production"),
```

Comment:
❓ probably not your creation, but why these types? I would have expected either differentiation like saas/ccsm or trial/production.